src/Integration/Binding/BindingController.cs

```diff
@@ -107,8 +107,8 @@ private bool OnBindStatus(BindCommandArgs args)
                    && !host.VisualStateManager.IsBusy
                    && (folderWorkspaceService.IsFolderWorkspace()
                        || (knownUIContexts.SolutionExistsAndFullyLoadedContext.IsActive
-                           && knownUIContexts.SolutionExistsAndNotBuildingAndNotDebuggingContext.IsActive))
-                   && (projectSystemHelper.GetSolutionProjects()?.Any() ?? false);
+                           && knownUIContexts.SolutionExistsAndNotBuildingAndNotDebuggingContext.IsActive
+                           && (projectSystemHelper.GetSolutionProjects()?.Any() ?? false)));
```

Comment:
`projectSystemHelper.GetSolutionProjects()` returns empty when it's a folder that is not CMake. The previous condition worked only for folder that is CMake.